frontend/src/server/index.js

```diff
@@ -0,0 +1,39 @@
+var app = require("express")();
```

Comment:
Now that I am looking at this, this doesn't seem right. You're using a server-side JS engine for presenting websites when in reality the MT server-side should be offering the html files. The client itself should be creating the websocket to the MT server, not to itself and THEN to the MT server.

Reply:
I understand now, thank you for the feedback. I will cancel this pull request